### ISSUE TITLE
DNS doesn't work in containers running on Ubuntu 22.04 host

### ISSUE THREAD

**Issue by jerkstorecaller** [89luca89/distrobox#851]
**Describe the bug**
NOTE: this seems to be similar to #810, except it applies to Ubuntu 16.04 and Debian 12 (and possibly more) containers.

My host is Ubuntu 22.04 running in a Virtualbox VM with default networking (ie DHCP). I have no problems running Docker containers in this VM.
I'm trying to run Ubuntu 16.04 in a container. 
I'm using distrobox 1.5.0 (latest, installed tonight using README's curl + sh instructions).

The problem is that DNS doesn't work at all in the containers.

The logs in "docker logs -f ubuntu-16.04" are full of:
```diff: /run/host../run/systemd/resolve/stub-resolv.conf: No such file or directory```

I suspect that dangling .. in the path is the issue (typo?), because the container CAN see the file if I use this path instead:
```
user@ubuntu-16:~$ stat /run/host/run/systemd/resolve/stub-resolv.conf 
  File: '/run/host/run/systemd/resolve/stub-resolv.conf'
  Size: 920             Blocks: 8          IO Block: 4096   regular file
```

**To Reproduce**
```
distrobox create --image ubuntu:16.04 --name ubuntu-16.04
distrobox enter ubuntu-16.04 #wait for installation
curl google.com
```

Result: ```curl: (6) Could not resolve host: google.com```

**Expected behavior**
DNS should work.

**Logs**
Docker logs are just  repeating
```
diff: /run/host../run/systemd/resolve/stub-resolv.conf: No such file or directory
````

[verbose.txt](https://github.com/89luca89/distrobox/files/12016858/verbose.txt)

**Desktop (please complete the following information):**
 - Docker version 24.0.4, build 3713ee1
 - Distrobox 1.5.0

**Additional context**
I also tried creating a new distrobox using debian:bookworm and it had the same issue.

**Comment by bubuntux** [comment]
have you tried using quay.io/toolbx-images/ubuntu-toolbox:22.04
or quay.io/toolbx/ubuntu-toolbox:22.04  ?

**Comment by jerkstorecaller** [comment]
I'm not sure I understand you. What do those Docker images have to do with my issue? I have a standard install of ubuntu 22.04, from the official ISO, which is my Linux desktop I work in. This is the most common Linux desktop.

**Comment by maxime915** [comment]
Hey @bubuntux , I'm running in the same DNS issues with the following images as guests

- quay.io/toolbx-images/ubuntu-toolbox:22.04
- quay.io/toolbx-images/ubuntu-toolbox:22.04
- ubuntu:22.04 (I suppose the one from dockerhub)

But not with registry.fedoraproject.org/fedora-toolbox:38

My host is a fresh Pop!_OS 22.04 LTS; distrobox: 1.5.0 (from the curl/wget install); Docker version 24.0.5, build ced0996; 6.2.6-76060206-generic

**Comment by 89luca89** [comment]
This has been fixed in 1.5.0.2 already
